support/filterlock/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[workspace]
+
+[package]
+name = "filterlock"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+serde = { version = "1.0.194", features = ["derive"] }
+toml = { version = "0.8.8", features = ["preserve_order"] }
```

Comment:
Would it be possible to describe what this tool does and explain that it's only used for nix here as well as adding a license.

Reply:
done in bd49d8f39ca58db6f618ed59fa810b99df392bc6 + 6f76a10fca2619f66182077031bc3430a1a4ba18